pkg/clusterapi/rollout_strategy.go

```diff
@@ -0,0 +1,37 @@
+package clusterapi
+
+import (
+	"k8s.io/apimachinery/pkg/util/intstr"
```

Comment:
nit: I think we have our own package for this

Reply:
I wasn't able to find the internal package for this, saw we're using intstr in a few other places including Tinkerbell. Know if it'd be somewhere outside utils?

Reply:
ignore me